src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -192,4 +201,140 @@ describe('LegacyConfigForm', () => {
+    userEvent.type(topicInput, 'valid-topic');
+    topicInput.blur();
+    await waitForElementToBeRemoved(queryByText(topic, 'Choose a unique name for your topic'));
+    expect(queryByText(topic, 'It looks like this name is already in use')).not.toBeInTheDocument();
+    expect(queryByText(topic, 'Topic name is a required field')).not.toBeInTheDocument();
+  });
+
+  test('Show duplicate errors on fields when passing duplicate topic name / ', async () => {
+    await mockStore(legacyApiResponse);
+    createComponent(defaultAppConfig);
+    const topicInput = createNewTopic('Edx');
+    const topic = topicInput.closest('.collapsible-card');
+    topicInput.blur();
+    await waitForElementToBeRemoved(queryByText(topic, 'Choose a unique name for your topic'));
+    expect(queryByText(topic, 'It looks like this name is already in use')).toBeInTheDocument();
+  });
+
+  test('check duplicate errors on fields when passing case insensitive duplicate topic name', async () => {
+    await mockStore(legacyApiResponse);
+    createComponent(defaultAppConfig);
+    const topicInput = createNewTopic('edx');
+    const topic = topicInput.closest('.collapsible-card');
+    topicInput.blur();
+    await waitForElementToBeRemoved(queryByText(topic, 'Choose a unique name for your topic'));
+    expect(queryByText(topic, 'It looks like this name is already in use')).toBeInTheDocument();
+  });
+
+  test('check duplicate error is removed on fields when name is fixed', async () => {
+    await mockStore(legacyApiResponse);
+    createComponent(defaultAppConfig);
+
+    const topicInput = createNewTopic('edx');
+    const topic = topicInput.closest('.collapsible-card');
+    topicInput.blur();
+    await waitForElementToBeRemoved(queryByText(topic, 'Choose a unique name for your topic'));
+    expect(queryByText(topic, 'It looks like this name is already in use')).toBeInTheDocument();
+
+    topicInput.setSelectionRange(0, topicInput.value.length);
+    userEvent.type(topicInput, 'valid');
+    expect(queryByText(topicInput, 'It looks like this name is already in use')).not.toBeInTheDocument();
+  });
+
+  test('check duplicate error is removed on deleting duplicate topic', async () => {
+    await mockStore(legacyApiResponse);
+    createComponent(defaultAppConfig);
+    const topicInput = createNewTopic('duplicate');
+    const topic = topicInput.closest('.collapsible-card');
+    const duplicateTopicInput = createNewTopic('duplicate');
+    const duplicateTopic = duplicateTopicInput.closest('.collapsible-card');
+    topicInput.blur();
+    duplicateTopicInput.blur();
+    await waitForElementToBeRemoved(queryByText(topic, 'Choose a unique name for your topic'));
+    expect(queryByText(topic, 'It looks like this name is already in use')).toBeInTheDocument();
+    expect(queryByText(duplicateTopic, 'It looks like this name is already in use')).toBeInTheDocument();
+    const deleteTopicButton = getByLabelText(duplicateTopic, 'Delete Topic');
```

Comment:
move this to its own action so that it can be reused. 
```
const deleteTopic = (topicCard) => {
    const deleteTopicButton = getByLabelText(topicCard, 'Delete Topic');
    userEvent.click(deleteTopicButton);
    expect(topicCard).not.toBeInTheDocument();
 
   const delConfirmButton = getByText(container, 'Delete');
    userEvent.click(delConfirmButton);
}
```